src/test/java/com/cloudbees/jenkins/plugins/sshagent/SSHAgentBuildWrapperTest.java

```diff
@@ -0,0 +1,55 @@
+package com.cloudbees.jenkins.plugins.sshagent;
+
+import hudson.model.FreeStyleProject;
+import hudson.tasks.Shell;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import com.cloudbees.jenkins.plugins.sshcredentials.SSHUserPrivateKey;
+import com.cloudbees.jenkins.plugins.sshcredentials.impl.BasicSSHUserPrivateKey;
+import com.cloudbees.plugins.credentials.CredentialsScope;
+import com.cloudbees.plugins.credentials.SystemCredentialsProvider;
+
+public class SSHAgentBuildWrapperTest extends SSHAgentBase {
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
+
+    @Before
+    public void starting () throws Exception {
+        startMockSSHServer();
+    }
+
+    @Test
+    public void sshAgentAvailable() throws Exception {
+        List<String> credentialIds = new ArrayList<String>();
+        credentialIds.add("84822271-02d5-47b8-b8ff-c40fef175c29");
```

Comment:
Huh? Why are you hardcoding what looks to be a random UUID but is not? Why not just define a simple constant?

```java
String id = "my-ssh-private-key";
```